modules/gapi/test/infer/gapi_infer_ie_test.cpp

```diff
@@ -56,6 +56,15 @@ class TestMediaBGR final: public cv::MediaFrame::IAdapter {
         cv::MediaFrame::View::Strides ss = { m_mat.step, 0u, 0u, 0u };
         return cv::MediaFrame::View(std::move(pp), std::move(ss), Cb{m_cb});
     }
+    cv::util::any blobParams() const override {
+        return std::make_pair<InferenceEngine::TensorDesc,
```

Comment:
I believe you don't have to specify template manually, `make_pair(...)` would be enough